src/main/scala/com/fullcontact/interview/RecordFinder.scala

```diff
@@ -1,7 +1,59 @@
+/*
+Author: FullContact
+Modified By: Jeff Willingham
+Creation Date: unknown
+Modification Date: 2020-07-28
+Purpose: This script inputs Queries.txt and Records.txt, performs transformations and writes outputs per specifications outlined in README.md.
+*/
+
 package com.fullcontact.interview
 
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
+import org.apache.spark.sql.functions._
+
 object RecordFinder {
   def main(args: Array[String]): Unit = {
-    // TODO: Implement your job here
+    val spark = SparkSession // instantiate spark session
+      .builder()
+      .appName("RecordFinder")
+      .master("local[*]")
+      .getOrCreate
+    val queriesDF: DataFrame = spark // read in Queries.txt as a DataFrame
+      .read
+      .format("text")
+      .load("Queries.txt")
+      .withColumnRenamed("value", "queryID")
+    val recordsDF: DataFrame = spark // read in Records.txt as a DataFrame, and create two fields
```

Comment:
Have you used RDDs in Spark before? If so I'm curious to hear your thoughts on how RDDs and DataFrames in terms of performance, ease of use/testing, etc.